plugins/inputs/logstash/logstash.go

```diff
@@ -290,6 +292,20 @@ func (logstash *Logstash) gatherPluginsStats(
 			return err
 		}
 		accumulator.AddFields("logstash_plugins", flattener.Fields, pluginTags)
+		if pluginType == "output" && plugin.Name == "elasticsearch" {
```

Comment:
I'm not very familiar with logstash, but wouldn't it make sense to _always_ add the `BulkRequests` and `Documents` information if they are available (i.e, non-nil)? Or does that only make sense for the configuration you test?

Reply:
They aren't consistently available. And these particular stats only exist for elasticsearch outputs from logstash.

Reply:
Ok, would be nice if you can add this information as a comment to the code.